capstone/src/main/java/com/google/sps/search/SearchPredictor.java

```diff
@@ -17,7 +17,10 @@
 
 public class SearchPredictor implements Serializable {
 
-  private static final int COMPLETE_PARTIAL_NAME_MATCH = 5;
+  private static final double COMPLETE_PARTIAL_NAME_MATCH = 5;
+  private static final double COMPLETE_MATCH = 10;
+  private static final double FIRST_NAME_MATCH = 2;
+  private static final double LAST_NAME_MATCH = 1;
```

Comment:
Can you add tests for comparative scoring? COMPLETE_MATCH result ranked above PARTIAL_NAME above FIRST above LAST

Reply:
An additional test could compare 0 edit distance, 1 edit distance, 2 edit distance, and if you wanted confirm that >= 3 edit distance does not return a result.